docs/en/05-supported-backends/rknn.md

```diff
@@ -0,0 +1,81 @@
+# RKNN support
+
+This tutorial is based on Linux systems like Ubuntu-18.04 and Rockchip NPU like `rk3588`.
+
+## Installation
+
+It is recommended to create a virtual environment for the project.
+
+1. get RKNN-Toolkit2 through:
+
+   ```
+   git clone https://github.com/rockchip-linux/rknn-toolkit2
+   ```
+
+2. install RKNN python package following [official doc](https://github.com/rockchip-linux/rknn-toolkit2/tree/master/doc).
+
+3. reinstall MMDeploy from source following the [instructions](../01-how-to-build/build_from_source.md). Note that there are conflicts between the pip dependencies of MMDeploy and RKNN. Here is the suggested packages versions for python 3.6:
+
+   ```
+   protobuf==3.19.4
+   onnx==1.8.0
+   onnxruntime==1.8.0
+   torch==1.8.0
+   torchvision==0.9.0
+   ```
+
+To work with models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/master/docs/get_started.md), you may need to install it additionally.
+
+## Usage
+
+Example:
+
+```bash
+python tools/deploy.py \
```

Comment:
Failed when running this command:
```
W __init__: rknn-toolkit2 version: 1.3.0-11912b58
Process Process-4:
Traceback (most recent call last):
  File "/home/PJLAB/lvhan/miniconda3/envs/mmdeploy-3.6/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/home/PJLAB/lvhan/miniconda3/envs/mmdeploy-3.6/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/PJLAB/lvhan/Documents/projects/open-mmlab/mmdeploy/mmdeploy/apis/core/pipeline_manager.py", line 107, in __call__
    ret = func(*args, **kwargs)
  File "/home/PJLAB/lvhan/Documents/projects/open-mmlab/mmdeploy/mmdeploy/backend/rknn/onnx2rknn.py", line 50, in onnx2rknn
    rknn.config(**common_params)
TypeError: config() got an unexpected keyword argument 'output_tensor_type'

```

Reply:
I used rknn-toolkit2 1.2.0-f7bb160f.

Reply:
> f7bb160f

Which commit id corresponds to `rknn-toolkit2 1.2.0-f7bb160f`? There is no 'f7bb160f' in its github repo by the way

Reply:
834ba0b0a1ab8ee27024443d77b02b5ba48b67fc